### ISSUE TITLE
Had an error while testing MLPClassifier

### ISSUE THREAD

**Issue by jspeed-meyers** [mlrequest/sklearn-json#1]
Great package! Thank you for creating this. I was testing out the MLPClassifier() model with this package--see the complete code below--and was getting an `AttributeError: 'list' object has no attribute 'ndim'` when I used the package as is. I added some code below to convert the model coefficient elements into numpy arrays and it seemed to work. Am I missing something? Was this an error with the package or was it an error in how I was using the package? Either way, thank you again. Note: I was using Python 3.7.4.

```python
import numpy as np
import sklearn_json as skljson
from sklearn.neural_network import MLPClassifier
from sklearn import datasets

# Load data
df = datasets.load_iris()
X = df.data 
y = df.target 

# Instantiate and fit model
model_fit = MLPClassifier(solver='lbfgs', alpha=1e-5,
	hidden_layer_sizes=(15,), random_state=1).fit(X, y)
file_name = "model.json"

# Convert model to json
skljson.to_json(model_fit, file_name)

# Convert json back to model
deserialized_model = skljson.from_json(file_name)

## THE MISSING PIECE: THE LISTS INSIDE COEFS_ NEED
## TO BE CONVERTED TO ARRAYS
for i, x in enumerate(deserialized_model.coefs_):
	deserialized_model.coefs_[i] = np.array(x)

# This prediction should run if the code really works
deserialized_model.predict(X)
```

**Comment by mathieujofis** [comment]
Hey thanks for the kind words and I'm glad you enjoy it. 

Do the tests pass for you? If so, then it's probably not your fault and just something I missed. Feel free to make a PR with an updated MLP classifier test.

**Comment by jspeed-meyers** [comment]
Thanks for consulting on this matter. There are three failing tests when I run pytest on the current codebase with my machine: test_mlp, test_svm, and test_svr. I am using Python 3.7.4, numpy 1.18.1, and scikit-learn 0.22. Perhaps this is an issue particular to one of these package versions.

**Comment by mathieujofis** [comment]
Yes, it is likely the new scikit-learn version, as this was developed with 0.21.3. I'm not as active with this project right now and should update the readme to reflect as such. I welcome contributions, however!
